Makefile

```diff
@@ -510,3 +510,14 @@ define run_engine_unit_test
 	|| { $(FAILPOINT_DISABLE); exit 1; }
 	$(FAILPOINT_DISABLE)
 endef
+
+engine_unit_test_in_verify_ci: check_failpoint_ctl tools/bin/gotestsum tools/bin/gocov tools/bin/gocov-xml
+	mkdir -p $(ENGINE_TEST_DIR)
+	$(FAILPOINT_ENABLE)
+	@export log_level=error; \
```

Comment:
not sure if we need some info level log to debug when CI fails

Reply:
Seems this environment variable doesn't affect the log behavior in unit test.